app/models/post.rb

```diff
@@ -1,22 +1,14 @@
 class Post < ApplicationRecord
-  has_many :comments
-  has_many :likes
+  has_many :comment
+  has_many :like
   belongs_to :author, class_name: 'User'
+  after_save :update_posts_counter
 
-  attribute :title, :string
-  attribute :text, :text
-  attribute :comments_counter, :integer, default: 0
-  attribute :likes_counter, :integer, default: 0
-
-  after_save :update_post_counter
-
-  private
-
-  def update_post_counter
-    author.increment!(:posts_counter)
+  def five_recent_comments
+    Comment.where(post_id: id).order(created_at: :desc).limit(5)
   end
 
-  def five_most_recent_comments
-    comments.order(created_at: :desc).limit(5)
+  def update_posts_counter
+    author.increment!(:posts_counter)
   end
```

Comment:
- [optional  ] Kindly make this method `update_posts_counter` private as suggested by the previous reviwer.